examples/kustomize/patch.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kustomize-test
+  labels:
+    app: kustomize-test
+spec:
+  replicas: 1
+  selector:
```

Comment:
Can you clear everything in here except the piece you're trying to patch? I think it's just image, but it would serve as a better example to reduce duplication.

Reply:
Yup will do, I lazy copy-pasta'd

Reply:
done